src/state/chain/updaters/AtomUpdater.tsx

```diff
@@ -57,9 +60,30 @@ const AtomUpdater = () => {
       mixpanel.register({
         wa: account.address,
       })
+      // Check if the wallet is a Safe Multisig
+      const checkIfSafe = async () => {
```

Comment:
move this outside of the useEffect like the timestamp fn, and only run it on a different useEffect that only checks for account?.address change, currently this re-run even if they change chains